qiskit/providers/__init__.py

```diff
@@ -118,22 +118,22 @@
 
 If you have a quantum device or simulator that you would like to integrate with
 Qiskit you will need to write a provider. A provider will provide Terra with a
-method to get available :class:`~qiskit.providers.BackendV1` objects. The
-:class:`~qiskit.providers.BackendV1` object provides both information describing
+method to get available :class:`~qiskit.providers.BackendV2` objects. The
+:class:`~qiskit.providers.BackendV2` object provides both information describing
 a backend and its operation for the :mod:`~qiskit.transpiler` so that circuits
 can be compiled to something that is optimized and can execute on the
-backend. It also provides the :meth:`~qiskit.providers.BackendV1.run` method which can
+backend. It also provides the :meth:`~qiskit.providers.BackendV2.run` method which can
 run the :class:`~qiskit.circuit.QuantumCircuit` objects and/or
 :class:`~qiskit.pulse.Schedule` objects. This enables users and other Qiskit
 APIs, such as :func:`~qiskit.execute_function.execute` and higher level algorithms in
-Qiskit Aqua, to get results from executing circuits on devices in a standard
-fashion regardless of how the Backend is implemented. At a high level the basic
+:mod:`qiskit.algorithms`, to get results from executing circuits on devices in a standard
+fashion regardless of how the backend is implemented. At a high level the basic
 steps for writing a provider are:
 
  * Implement a :class:`~qiskit.providers.ProviderV1` subclass that handles
    access to the backend(s).
- * Implement a :class:`~qiskit.providers.BackendV1` subclass and its
-   :meth:`~~qiskit.providers.BackendV1.run` method.
+ * Implement a :class:`~qiskit.providers.BackendV2` subclass and its
+   :meth:`~~qiskit.providers.BackendV2run` method.
```

Comment:
missing `.` (I can't seem to suggest changes in single-commit view.)

Reply:
Fixed in the current revision (just forced pushed as part of a rebase too)